ui/apps/ui/src/app/layouts/secondary-tags/secondary-tags.component.ts

```diff
@@ -78,6 +84,48 @@ export class SecondaryTagsComponent implements OnChanges {
   @Output()
   activeFilter = new EventEmitter<{ filter: string; value: string }>();
 
+  arrayOfTags: IValueWithLabel[] = [];
+  show = false;
+  temp: IValueWithLabel[] = [];
+
+  constructor(private cdr: ChangeDetectorRef) {}
+
+  ngAfterViewInit() {
+    this.cdr.detectChanges();
+  }
+
+  ngAfterContentChecked(): void {
+    this.cdr.detectChanges();
```

Comment:
smart but.. it wont call onChanges, onChanges called only when inputparams changed, even if you call detectChanges still onChanges not called, i would still implement onChanges interface.. Please double check if what you did here is intentional or a mistake ;)

Reply:
I made a mistake. Old unused code